### ISSUE TITLE
Responsive text utilities are missing the !important rule (v4)

### ISSUE THREAD

**Issue by asnoba** [twbs/bootstrap#18300]
When doing something like `<p class="text-left text-md-right">I'm always aligned to the left</p>` the `text-left` helper class takes precedence, because the responsive utility classes are missing the `!important` rule (or should be removed from the base classes):

```
.text-left           { text-align: left !important; }
.text-right          { text-align: right !important; }
.text-center         { text-align: center !important; }
.text-justify        { text-align: justify !important; }
.text-nowrap         { white-space: nowrap !important; }
.text-truncate       { @include text-truncate; }

// Responsive alignment

@each $breakpoint in map-keys($grid-breakpoints) {
  @include media-breakpoint-up($breakpoint) {
    .text-#{$breakpoint}-left   { text-align: left; }
    .text-#{$breakpoint}-right  { text-align: right; }
    .text-#{$breakpoint}-center { text-align: center; }
  }
}
```

**Comment by cvrebert** [comment]
Fixed by #18314.

**Comment by Xerotherm1c** [comment]
`<p class="text-left text-md-right">Responsive text-md-right works</p>`
^ since [#18314](https://github.com/twbs/bootstrap/pull/18314/commits/daa319067830b694118a5de45e7be51e5e4e0b7d)

But
`<p class="text-left text-md-justify">Responsive text-md-justify does not exist</p>`

Is there a reason responsive justify classes appear to have been left out?
